src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$page-color: #b0c4de;
+$animation-hour-time: 86400s;
```

Comment:
The hour hand makes a full rotation in 12 hours, which equals to 12 hours x 60 minutes per hour x 60 seconds per minute = 43,200 seconds